<launch>
 <node pkg="mission_pkg" type="bt_ros" name="mission" output="screen">
    <param name="is_gen_xml" type="bool" value="true"/>
    <param name="xml_file" type="str" value="$(find mission_pkg)/config/all_node.xml"/>

 </node>
</launch>
